src/sql/src/session/vars.rs

```diff
@@ -874,6 +874,17 @@ pub const ENABLE_SESSION_RBAC_CHECKS: ServerVar<bool> = ServerVar {
     internal: false,
 };
 
+/// Whether compute rendering should use the new linear join implementation rather than the one
+/// from Differential Dataflow.
+const ENABLE_NEW_LINEAR_JOIN_IMPLEMENTATION: ServerVar<bool> = ServerVar {
```

Comment:
A nit, and not to be solved in this PR. We should figure out better names for our feature flags. `NEW_LINEAR_JOIN` does not convey anything about the nature of the change, and ideally it should be clear from the description what it does. Something that's new today will be old tomorrow :D

Reply:
Well, the idea is to arrive at a single join implementation and remove that flag before the new one becomes old :) But I also don't like the name. Ideally we'd have one that conveys that a) the new join has better yielding behavior than the DD join and b) the new join is inferior for workloads with fine-grained times. But I can't come up with anything good.

We could also side-step the issue, invert the meaning of the flag and call it `enable_dd_linear_join_implementation`. But then we'd have a feature flag that needs to be set to `false` to enable the feature it protects, which would be confusing.

Reply:
How about `enable_mz_join_core`?

Reply:
Thanks, that seems like a better name. Changed.